_posts/2015-07-08-employee-induction-programme.md

```diff
@@ -0,0 +1,96 @@
+
+* information about the starting date and time of the first workday
+* an induction programme giving some indication of what will happen on their first days at ePages
+* a personnel record that's to be completed by the day the employee starts with us
+
+# 2. Involve teams and organise buddies
+
+After all, a new developer will be part of a development **team**.
+We spread out the induction plan to the team to have them informed about the new colleague being in which training, why and when.
+This will allow the team to organise and plan their tasks accordingly and coordinate the induction sessions and pair programming with the new team member upfront.
+Of course, this also includes setting up the workspace and ordering IT equipment.
+We furthermore involve colleagues from other departments to volunteer as mentors for the new employees.
+Here at ePages we call them "buddies".
+
+# 3. First work day: Meet with new employees right away
+
+When having arrived at the office, HR welcomes the new employees, a round of introductions follows and the further course of the day will be explained.
+
+{% image blog-employee-induction-CTO.jpg %}{% endimage %}
+
+Our CTO gives a brief introduction including:
+
+* Development teams and their tasks
+* Goals and targets
+* Software architecture
+* Benefits for our merchants
+
+# 4. Buddy time!
+
+Get the newbies into contact with their buddies and off they go exploring the office.
+This is all about settling-in and clarifying feel-good issues as:
+
+* Coffee machine
+* Rest rooms
+* Kitchen use
+* Storage areas
+* Office systems
+* Social rooms
+
+Buddies show the layout of the offices and the new workplace, introduce the new employees to staff of other departments and may answer any questions that the employee is not comfortable posing to the team or supervisor.
+They'll be shown their workplaces as well as important contact points.
+
+# 5. Introduction session with HR
+
+{% image blog-employee-induction-welcome.jpg %}{% endimage %}
+
+Our HR team then hands out the office keys and local transport tickets and familiarises our new employees.
+This includes:
+
+* Orga chart
+* Company values
+* Benefits
+* Documents to be submitted
+* Travel guidelines
+* Target agreement discussions
+* Company events and holidays
+* Applications for leave and sick leave
+
+# 6. Team lunch
+
+Phew! It's definitely time for a break and getting to know the team members during lunch.
+
+# 7. At the desk with the team
+
+Now it's team time: Setting up the workplace, configuring the laptop, collecting office materials and receiving more "getting started" information from the team.
+
+# And what's next?
+
+It's most probably information overkill.
+The first day is over and the new colleagues have lot's of new information in their luggage.
+The agenda for the following days contains some more interesting topics and valuable tipps for a good start at ePages.
+
+{% image blog-employee-induction-agenda.png %}{% endimage %}
```

Comment:
Comments concerning the picture: It starts with day 2, intentionally? Do we really want to show this detailed information about this internal process? In my opinion it's good and interesting. But maybe we should confirm this with Harm (if that hasn't been done already).